experimental/embeddings/scripts/evaluate_promises_sequence_tagging.py

```diff
@@ -46,19 +44,19 @@ def evaluate_sequence_tagging(
     embeddings = TransformerWordEmbeddings(embedding)
 
     print("Training model...")
-    tagger = st.FlairSequenceTagger(
-        embeddings=embeddings,
-        hidden_dim=256,
-        tag_dictionary=tag_dictionary,
-        output_path=str(out_dir),
-    )
-
-    training_log = tagger.fit(corpus)
-    evaluation_log = tagger.evaluate(test_sentences)
-    with out_dir.joinpath("results.pkl").open(mode="wb") as f:
-        pickle.dump(obj={"training_log": training_log, "evaluation_log": evaluation_log}, file=f)
-
-    print("Done!", training_log)
+    # tagger = FlairSequenceTagger(
```

Comment:
why commented?

Reply:
this is legacy file. FlairSequenceTagger was removed. promise elections dataset should be framed and moved from `experimenal`. it is on my todo list and I'm going to do that on a separated PR